.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "thirdparty/account-abstraction"]
+	path = thirdparty/account-abstraction
+	url = https://github.com/eth-infinitism/account-abstraction.git
```

Comment:
@Vid201 I am using submodules to keep track of the interfaces here. I am not sure whether you prefer submodule or subtree. I am fine with either way. Please tell me what you prefer.

Reply:
submodules are great!